lib/Core/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ endif()
 
 
 klee_get_llvm_libs(LLVM_LIBS ${LLVM_COMPONENTS})
-target_link_libraries(kleeCore PUBLIC ${LLVM_LIBS})
+target_link_libraries(kleeCore PUBLIC ${LLVM_LIBS}  ${SQLITE3_LIBRARIES})
```

Comment:
Wouldn't that add sqlite to kleaver?

Reply:
Probably, but where would you put it to not do it? I don't want to move `StatsTracker` out of core

Reply:
And leave kleaver without sql references? If kleeCore gets into kleaver it's not my fault.

Reply:
Then keep it.